src/libxtp/pmdecomposition.cc

```diff
@@ -0,0 +1,122 @@
+#include "votca/xtp/pmdecomposition.h"
```

Comment:
Add license

Reply:
Okay wait...I think I messed up a little with the branches. New user mistakes. This thing was supposed to be comitted into the decomptool branch. However I will incorporate these suggestions there

Reply:
No problem, if you need help let me know.